<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Définition des actions</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sur cette fenêtre, vous pouvez définir les actions de votre séquence une par une.&lt;/p&gt;&lt;p&gt;1. Utilisez le menu déroulant pour choisir le type d'action souhaité ;&lt;/p&gt;&lt;p&gt;2. Entrez la durée souhaitée. Attention : elle doit être supérieure ou égale à 12 secondes ;&lt;/p&gt;&lt;p&gt;3. Cliquez sur &amp;quot;Valider&amp;quot; ou appuyez sur la touche Entrée ;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Informations sur l'action :</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Nom de l'action :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="text_1">
            <property name="styleSheet">
             <string notr="true">
font: 63 14pt &quot;Segoe UI Semibold&quot;;</string>
            </property>
            <item>
             <property name="text">
              <string>Total broth</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Surnageant</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Intracellulaire</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Durée de l'action (en secondes) :</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="text_2">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bouton_valider">
            <property name="styleSheet">
             <string notr="true">
font: 63 14pt &quot;Segoe UI Semibold&quot;;</string>
            </property>
            <property name="text">
             <string>Valider</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bouton_valider</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>valider_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valider_action()</slot>
 </slots>
</ui>
